varya/assets/js/customizer-validate-wcag-color-contrast.js

```diff
@@ -0,0 +1,214 @@
+/**
+ * @author Per Soderlind
```

Comment:
Would you mind adding license info for this to the [`readme.txt`](https://github.com/Automattic/themes-workspace/blob/b9fcb3331d76efd942b9e553ca91557bf3523494/varya/readme.txt#L44) and the [`style.css`](https://github.com/Automattic/themes-workspace/blob/b9fcb3331d76efd942b9e553ca91557bf3523494/varya/style.css#L36) header?